docs/rooms/fireRoom.md

```diff
@@ -0,0 +1,59 @@
+# Fire Room
+
+A room with three different paths.
+One path leads to a person, the second path leads to a item and the third path leads no where.
```

Comment:
nit: Should it be "nowhere" instead of "no where"

Reply:
updated